Neos.Neos/Classes/Service/Controller/AbstractServiceController.php

```diff
@@ -68,35 +71,35 @@ public function errorAction()
      * @param RequestInterface $request The request object
      * @param ResponseInterface $response The response, modified by this handler
      * @return void
-     * @throws \Exception
+     * @throws Exception
      */
     public function processRequest($request, $response)
```

Comment:
The signature is now `processRequest(ActionRequest $request, ActionResponse $response)`